presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -245,17 +256,19 @@ private static AWSCredentialsProvider getCustomAWSCredentialsProvider(String pro
     public List<String> getAllDatabases()
     {
         try {
-            List<String> databaseNames = new ArrayList<>();
-            String nextToken = null;
+            return stats.getGetAllDatabases().wrap(() -> {
+                List<String> databases = new ArrayList<>();
```

Comment:
unrelated change (used to be `databaseNames`). This can stay, but generally please avoid unrelated changes